src/components/common/Modal/modal.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Modal from 'react-modal';
+import { PropTypes as PT } from 'prop-types';
+import container from './container';
+
+if (process.env.NODE_ENV !== 'test') Modal.setAppElement('#root');
```

Comment:
How come we need to do this?

Reply:
@girwin90 this is a fix for a react-modal error that is being thrown in the tests https://github.com/reactjs/react-modal/issues/632